bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -251,9 +251,13 @@ public HTableDescriptor getTableDescriptor(TableName tableName)
 
     try {
       return tableAdapter.adapt(bigtableTableAdminClient.getTable(request));
+    } catch (StatusRuntimeException e) {
+      if (e.getStatus().getCode() == Status.NOT_FOUND.getCode()) {
```

Comment:
fine with checking code too, thought not found status should be the static instance: public static final Status NOT_FOUND.